src/client/uimap.js

```diff
@@ -13,6 +14,45 @@ UIMap.initialize = async function() {
   const basic = await WZManager.get('UI.wz/Basic.img');
   this.statusBarLevelDigits = basic.LevelNo.nChildren.map(d => d.nGetImage());
 
+  this.chat = new MapleInput({
+    x: 5,
+    y: 540,
+    width: 530,
+    color: '#000000',
+    background: '#ffffff',
+    height: 13
+  });
+  this.chat.input.style.zIndex = 4;
```

Comment:
You should not be setting the z index for input elements. Please have a look at https://github.com/johncintron/nodin/blob/master/client/all.css#L31. Z index of 4 is reserved for the cursor, as seen here https://github.com/johncintron/nodin/blob/master/src/client/uicommon.js#L41. This is to ensure that the cursor appears above the input element.